http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -75,8 +77,6 @@ class ClusterTable(tag: Tag) extends Table[ClusterRecord](tag, "CLUSTER") {
   def customClusterEnvironmentVariables = column[Option[Map[String, String]]]("customClusterEnvironmentVariables")
   def deletedFrom = column[Option[String]]("deletedFrom")
 
-  def uniqueKey = index("IDX_CLUSTER_UNIQUE", (googleProject, clusterName, destroyedDate), unique = true)
```

Comment:
Can we keep this `def`? I'm not sure what it actually does though..

Reply:
based on doc in slick, I think this just adds the unique index, which is duplicated by [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20171010_cluster_unique_index.xml#L5)....